client/state/comments/test/reducer.js

```diff
@@ -135,6 +136,23 @@ describe( 'reducer', () => {
 			expect( result[ '1-1' ][ 0 ].placeholderState ).to.equal( PLACEHOLDER_STATE.ERROR );
 			expect( result[ '1-1' ][ 0 ].placeholderError ).to.equal( 'error_message' );
 		} );
+
+		it( 'should edit a comment by id', () => {
+			const state = deepFreeze( {
+				'1-1': [ { ID: 123, text: 'lorem ipsum' } ],
```

Comment:
should `text` here be `content`?

Reply:
![that is correct](https://user-images.githubusercontent.com/233601/29854229-fb90674e-8d1a-11e7-8712-4a82044be122.gif)